### PR TITLE
[Tgbot 290] Create http client user gateway implementation

### PR DESCRIPTION
**Pull request by Artem-Metlushko**
https://schneidervs.atlassian.net/browse/TGBOT-290

### CODE REVIEW

**Review by max-java — CHANGES_REQUESTED** [review]
ptal with https://github.com/Tutritp01/TgBot_WebClient/pull/21

File: http-client/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+endpoint.web-client=http://localhost:8080
```
Comment (Sav4ykVladzislau) [inline http-client/src/main/resources/application.properties]: it wasn't supposed to get here.

**Review by yurasava — DISMISSED** [review]

**Review by yurasava — DISMISSED** [review]

File: http-client/src/main/resources/application.properties (hunk 1 above)
Comment (Artem-Metlushko) [reply http-client/src/main/resources/application.properties]: I've deleted

**Review by AlexanderYarokhovich — DISMISSED** [review]

**Review by schneidervs — DISMISSED** [review]

**Review by yurasava — APPROVED** [review]

**Review by schneidervs — APPROVED** [review]

**Review by AlexanderYarokhovich — APPROVED** [review]

**Review by schneidervs — APPROVED** [review]

File: http-client/src/main/java/com/tutrit/httpclient/HttpUserGateway.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            throw new RuntimeException("Can't read mapJsonStringToUser", e);// TODO: 07.03.2023
+        }
+    }
+
+    private HttpResponse<String> sendHttpRequest(final HttpRequest request) {
+        try {
+            return httpClient.send(request, HttpResponse.BodyHandlers.ofString());
+        } catch (IOException | InterruptedException e) {
+            Thread.currentThread().interrupt();
+            throw new RuntimeException("Can't read getStringHttpResponse", e);// TODO: 07.03.2023
```
Comment (max-java) [inline http-client/src/main/java/com/tutrit/httpclient/HttpUserGateway.java:72]: todo what?

**Review by Sav4ykVladzislau — APPROVED** [review]

**Review by max-java — APPROVED** [review]
lgfm

**Pull request merged**